.travis.yml

```diff
@@ -14,4 +14,3 @@ install:
 
 script:
   - flake8
-  - pylint ksoftapi
```

Comment:
Why? Pylint can pick up things flake8 doesn't and vice-versa. flake8 seems more useful for "picking up "garbage" (e.g. extraneous whitespace) while pylint seems to cover both garbage detection and also code smells (invalid syntax/cyclic imports etc).